src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs

```diff
@@ -929,13 +929,13 @@ private bool IsDistributedTracingEnabled()
             EnvironmentOverrides(_localConfiguration.distributedTracing.account_id, "NEW_RELIC_ACCOUNT_ID")
             : _serverConfiguration.AccountId;
 
-        public int? SamplingTarget => _serverConfiguration.SamplingTarget;
+        public int? SamplingTarget => ServerlessModeEnabled ? 10 : _serverConfiguration.SamplingTarget;
```

Comment:
Should this be more like the following to avoid the conditional check with each access?
```suggestion
        public int? SamplingTarget { get; } = ServerlessModeEnabled ? 10 : _serverConfiguration.SamplingTarget;
```

Reply:
We use this same pattern in about a zillion places already; I can't imagine there'll be much of a performance improvement by making this change.

Reply:
The main reason I bring it up is that I believe that Sampling is invoked more frequently than other settings.

Reply:
`AdaptiveSampler` stores the current value of `SamplingTargetPeriodInSeconds` and `SamplingTarget` anytime there's a configuration change. So this property is only referenced once at a configuration change.